### PR TITLE
add esercizi backend fuzzyWuzzy

### PR DESCRIPTION
**Pull request by rastellid**

### CODE REVIEW

File: esercizi/fizzyWuzzy/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+## 👨🏻‍🏫 Requisiti
+___
+
+Scrivere un algoritmo che dato in input un serie di numeri positivi (maggiori di 0):
+
+1.  Ritornano una stringa `Fizz` se il numero è multiplo di `3`;
+2.  Ritornano una stringa `Wuzzy` se il numero è multiplo di `5`;
+3.  Ritornano una stringa `FizzWuzzy` se il numero è multiplo di `7`;
```
Comment (gioboa) [inline esercizi/fizzyWuzzy/README.md:19]: is it the right name? 😄

Comment (gioboa) [reply esercizi/fizzyWuzzy/README.md:19]: the folder name esercizi/fizzyWuzzy is wrong as well

**Review by gioboa — CHANGES_REQUESTED** [review]

File: esercizi/fizzyWuzzy/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+1. Da console eseguire il comando ``make build``
+2. Completata la build dell'immagine eseguire il comando ``make run`` e successivamente eseguire ``composer install``
+
+
+## 👨🏻‍🏫 Requisiti
+___
+
+Scrivere un algoritmo che dato in input un serie di numeri positivi (maggiori di 0):
+
+1.  Ritornano una stringa `Fizz` se il numero è multiplo di `3`;
```
[the remark this answers is not in the document]
Comment (rastellid) [reply esercizi/fizzyWuzzy/README.md]: 🤦🏻‍♂️ right, thanks 🙇🏻‍♂️

File: esercizi/fizzyWuzzy/README.md (hunk 1 above)
Comment (rastellid) [reply esercizi/fizzyWuzzy/README.md]: 🤦🏻‍♂️ right, thanks 🙇🏻‍♂️

**Review by gioboa — APPROVED** [review]

**Pull request merged**